code/processes/tailreader.q

```diff
@@ -25,13 +28,9 @@ reload:{
   @[.Q.l ;.tr.taildir;{.lg.e[`load;"Failed to load intradayDB with error: ",x]}];
   .lg.o[`load;"intradayDB loaded"];
   .lg.o[`load;"loading accesstable"];
-  .ds.access:@[.Q.l;accesstabdir;{.lg.e[`load;"Failed to load tailer accesstable with error: ",x]}];
-  /- select last set of entries from accesstable
-  .ds.access:select by table from .ds.access;
+  /- make a connection to the tailer to get the in-memory access table
+  tailerhandle: first exec w from .servers.getservers[`proctype;.tr.tailertypes;()!();1b;0b];
+  .ds.access:tailerhandle".ds.access";
```

Comment:
should probably error trap this & think about how we handle scenario where we can't retrieve access table for whatever reason

I think we should also do a check before this line to make sure we got a valid handle, and log an error if not